src/base64.h

```diff
@@ -60,7 +60,7 @@ bool base64_decode_group_slow(char* const dst, const size_t dstlen,
                               size_t* const i, size_t* const k) {
   uint8_t hi;
   uint8_t lo;
-#define V(expr)                                                               \
+# define V(expr)                                                            \
```

Comment:
nit: make sure the continuation `\` at the end is lined up. It might make sense to actually check for those in linting also if possible to do so reasonably.

Reply:
Thanks for finding that, I thought I had gotten them all. Updated all the files now